lib/graph/helpers/ui5Framework.js

```diff
@@ -339,20 +348,26 @@ export default {
 			Resolver = (await import("../../ui5Framework/Sapui5Resolver.js")).default;
 		}
 
+		// ENV var should take precedence over the dataDir from the configuration.
+		let ui5DataDir = process.env.UI5_DATA_DIR;
+		if (!ui5DataDir) {
+			const config = await Configuration.fromFile();
+			ui5DataDir = config.getUi5DataDir();
+		}
+		if (ui5DataDir) {
```

Comment:
what happens if ui5DataDir is nullish?

Reply:
Then that is passed to the API and should be handed the same as if it's undefined.